pkg/storage/storage.go

```diff
@@ -184,6 +186,11 @@ type ChangelogBackend interface {
 	ReadChanges(ctx context.Context, store, objectType string, paginationOptions PaginationOptions, horizonOffset time.Duration) ([]*openfgav1.TupleChange, []byte, error)
 }
 
+type Reader interface {
```

Comment:
Entities we have in FGA:

* Stores
* Assertions
* Models
* Tuples
* Changelog Entries (changes)

`Reader` is ambiguous and not specific enough. `ModelAndTupleReader` maybe?